src/scss/dp/overrides/components/_search.scss

```diff
@@ -53,13 +62,14 @@
     font-family: $base-font-family;
     font-weight: inherit;
     line-height: $base-line-height;
-    font-size: 17px;
+    font-size: 18px;
```

Comment:
here the same ... consider using relative font size value for accessibility ?

Reply:
fixed